docs/src/main/paradox/release-notes/index.md

```diff
@@ -1,6 +1,15 @@
 # 0. Release Notes
 
+## 1.0.1
+
+This is a bug fix release. There have also been some doc and build improvements.
```

Comment:
Need a diff link or a milestone to see what's changed.

Reply:
we haven't provided that in any previous set of release notes

Reply:
there is no release tag yet to use for a diff but users can use GitHub Compare for themselves

Reply:
I have added #443 - this matches what we have in https://github.com/apache/incubator-pekko/blob/main/CHANGELOG.md